dev/dev-env-setup.sh

```diff
@@ -130,7 +130,7 @@ if [ -z "$pyenv_exist" ]; then
       sudo apt-get update -y
       sudo apt-get install -y make build-essential libssl-dev zlib1g-dev \
       libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
-      libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+      libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev pandoc \
```

Comment:
what's the version installed by apt-get?

Reply:
yea it's older than 2, just found these lines: https://github.com/mlflow/mlflow/blame/master/mlflow/R/mlflow/Dockerfile.dev#L14-L18

Reply:
We may want to write it down that, indeed I've been using pandoc 1.19.2 in devbox without notice... (tho I ususually build on Mac)

Reply:
Can we use `wget` and `dpkg` to make sure pandoc is not old?

Reply:
yup, just pushed the new revision with updated script:)